.azure_pipelines/job_templates/olive-example-template.yaml

```diff
@@ -5,13 +5,14 @@ parameters:
   pool: ''
   python_version: '3.8'
   onnxruntime: 'onnxruntime'
+  subfolder: ''
```

Comment:
the default could be `local`. Or else, this PR also needs to update the stable and nightly pipelines also.

Reply:
Sounds good. Updated.